Gemfile

```diff
@@ -14,7 +14,8 @@ gem 'seamless_database_pool'
 
 gem 'le', '~> 2.2'
 gem 'os'
-gem 'redis', '~> 3.1.0'
+gem 'redis', '~> 3.3.1'
+gem 'redis-slave-read', require: false, github: 'islemaster/redis-slave-read', ref: 'master'
```

Comment:
Upstream [merged  my fix](https://github.com/cheald/redis-slave-read/pull/2) but hasn't published a new version yet.  Depending on my fork until they do so.  We should definitely switch back when we can.